packages/react-components/src/input/src/wrappedInputPropsAdapter.ts

```diff
@@ -0,0 +1,29 @@
+import { UseFieldInputPropsReturn, useFieldInputProps } from "../../field";
+
+export interface FieldWrappedInputProps extends Omit<UseFieldInputPropsReturn[0], "className"> {
+    wrapperProps: {
+        className: string;
+    };
+}
+
+export function useFieldWrappedInput(): [FieldWrappedInputProps, boolean] {
+    const [{ className, ...fieldProps }, isInField] = useFieldInputProps();
+
+    const props = {
+        ...fieldProps,
+        wrapperProps: {
+            className
+        }
+    };
+
+    return [props, isInField];
+}
+
+export function wrappedInputPropsAdapter({ className, ...rest }: Record<string, any>) {
```

Comment:
I believe it should have the same return type as useFieldWrappedInput (minus the array of course)

Reply:
should
```

export function useFieldWrappedInput(): [FieldWrappedInputProps, boolean] {
    const [{ className, ...fieldProps }, isInField] = useFieldInputProps();

    const props = {
        ...fieldProps,
        wrapperProps: {
            className
        }
    };

    return [props, isInField];
}
```

become 
```

export function useFieldWrappedInput(): [FieldWrappedInputProps, boolean] {
    const [fieldProps, isInField] = useFieldInputProps();

    const props =wrappedInputPropsAdapter(fieldProps);

    return [props, isInField];
}
```

Reply:
Sounds good